docs/UserGuide.md

```diff
@@ -79,6 +79,32 @@ Example:
 * `add i/A0123456Z n/john m/CS2103T p/98765432 t/@john e/E0123456Z`
     * A student named John is added to TAPA.
 
+### Deleting a student: `delete`
+
+Deletes a student from TAPA.
+
+Format: `delete STUDENT_INDEX`
+
```

Comment:
Perhaps we can include details about invalid index? (like maybe we can mention that the specified index must be less than or equals to the number of entries in TAPA)